pages/downloads.md

```diff
@@ -85,26 +85,29 @@ available in a future update to this site.
 {% for srcbin in srcbinArray %}
 {% assign lnkFile = 'accumulo-' | append: linkVers | append: '-' | append: srcbin | append: '.tar.gz' %}
 {% assign lnkSuffix = '/accumulo/' | append: linkVers | append: '/' | append: lnkFile %}
-<div class="row btn-group" style="margin-left: 20px; margin-bottom: 5px; display: block">
-  <div class="col btn-group">
+<div class="row" style="margin-left: 20px; margin-bottom: 5px;">
+  <div class="btn-group col-auto">
     <a {{btnDownloadStyle}} href="{{closerLink}}{{lnkSuffix}}" link-suffix="{{lnkSuffix}}">{{lnkFile}}{{glyphSave}}</a>
-  </div><div class="col btn-group">
-    <a {{btnSigStyle}} href="{{downloadsLink}}{{lnkSuffix}}.asc">ASC{{glyphLock}}</a>
-    <a {{btnHashStyle}} href="{{downloadsLink}}{{lnkSuffix}}.sha512">SHA{{glyphLock}}</a>
+    <div class="btn-group">
+      <a {{btnSigStyle}} href="{{downloadsLink}}{{lnkSuffix}}.asc">ASC{{glyphLock}}</a>
+      <a {{btnHashStyle}} href="{{downloadsLink}}{{lnkSuffix}}.sha512">SHA{{glyphLock}}</a>
+    </div>
```

Comment:
This still looks substantially different from what it was before, and I'm not sure if it has the same behavior as before. Originally, it was two button groups adjacent to one another, as if two columns in the same row. Here, you've changed it to a button group nested inside another, and the row effectively has one column now (which obviates the value of having a row).

I think it should be:

```suggestion
  </div><div class="btn-group">
      <a {{btnSigStyle}} href="{{downloadsLink}}{{lnkSuffix}}.asc">ASC{{glyphLock}}</a>
      <a {{btnHashStyle}} href="{{downloadsLink}}{{lnkSuffix}}.sha512">SHA{{glyphLock}}</a>
```

Where this matters is when you make the window size very small. This makes the buttons wrap in a very specific way that keeps the validation buttons (ASC, SHA) together, and situated just beneath the link to the file (rather than forcing the user to scroll), but without allowing them to easily wrap, creating a column of buttons mangled together with no logical groupings at all.

Reply:
Should be fixed as of 531aded.

Here is a screencap with only the top one fixed to show the difference:
![new button wrap](https://github.com/apache/accumulo-website/assets/47725857/6e25343b-bddd-4335-9626-33bbc1379982)

And here is a screencap with all fixed (how it appears with the commit above):
![whole page wrap 5](https://github.com/apache/accumulo-website/assets/47725857/9bb083a9-e053-4e50-87a0-e39ac7ef657a)

Reply:
The old buttons were lighter, and had mouse-over effects that made them slightly darker as a highlight before you click them. I'm not sure if the new ones have that... but one thing that is clearly different is that the ASC and SHA buttons seem to blend together. In the old version, they had a clear border, so you could tell they were separate buttons. We may need to fix the padding for those.

Reply:
One other thing I noticed is the lock icons probably need a `&nbsp;` between ASC and the icon and between SHA and the icon, to prevent the icon from wrapping on its own. It should stay with the text.

Reply:
Oh, I found the colors do have a lighter variant that look a bit better. They are the `-subtle` versions. I don't see subtle options for buttons, only for badges. I thought maybe you could get away with just using `btn bg-primary-subtle`, for example, instead of `btn btn-primary`, but the badges don't have the hover action like buttons. So, we'd just have to override the colors for the button classes in our own CSS to the same as what the subtle versions are, if we want to use the lighter colors and still have a mouseover effect that tints them slightly darker.

Reply:
> The old buttons were lighter, and had mouse-over effects that made them slightly darker as a highlight before you click them.

Here is how the new buttons behave:
![button hover](https://github.com/apache/accumulo-website/assets/47725857/bdb3715d-a7b6-4bd9-8992-5c4d2a3114d5)

> the ASC and SHA buttons seem to blend together. In the old version, they had a clear border

I noticed this as well but that seems to be the new bootstrap default. There are classes like `btn-outline-secondary` that have a light background and the outline and text is of the secondary color (see example below) but not sure if thats something we want to do. We could also override the button styles as you suggested in the other comment.
![image](https://github.com/apache/accumulo-website/assets/47725857/282fc4fd-3668-467c-8ab4-142d49ee92c6)

> the lock icons probably need a &nbsp; between ASC and the icon and between SHA and the icon

This is how the code is now in this branch. `{% assign glyphSave = '&nbsp;<span class="fa-solid fa-cloud-arrow-down"></span>' %}`

Reply:
Oh, I like the outline variant. Let's go with that one. Much easier to not have to mess with colors. I still think we need to add some padding between the button groups, though. They're a bit mashed up against one another. The spacing between the bottom row (release notes, readme, etc.) are nice. It'd be nice to have the same between the download links and the checksum buttons. The line between the ASC/SHA buttons are sufficient, though, so we don't need a space there... it seems the style as a "button group" there is fine (whereas I was previously thinking it as a "group of buttons" rather than a "button group").

Reply:
> This is how the code is now in this branch. `{% assign glyphSave = '&nbsp;<span class="fa-solid fa-cloud-arrow-down"></span>' %}`

That's for the arrow for the download link. I assume there's a similar for the ASC/SHA, but it didn't look like it worked in your visualization. Not sure why.

Reply:
> > This is how the code is now in this branch. `{% assign glyphSave = '&nbsp;<span class="fa-solid fa-cloud-arrow-down"></span>' %}`
> 
> That's for the arrow for the download link. I assume there's a similar for the ASC/SHA, but it didn't look like it worked in your visualization. Not sure why.

`{% assign glyphLock = '&nbsp;<span class="fa-solid fa-lock"></span>' %}`. The lock icon code has it too. Which visualization doesn't look right?

Reply:
In bad8a2d I added a slight margin to the right of the download button and made the other two buttons the outline style:
![Screenshot from 2023-09-12 14-33-21](https://github.com/apache/accumulo-website/assets/47725857/77da3b24-eef6-40ac-b9b5-f58357d9dc2d)
And when wrapping:
![Screenshot from 2023-09-12 14-33-35](https://github.com/apache/accumulo-website/assets/47725857/e4485e5b-22db-462b-a71a-9afa7cb31c95)

Alternatively, here is what it would look like for the download button to have the outline style too:
![Screenshot from 2023-09-12 14-34-19](https://github.com/apache/accumulo-website/assets/47725857/d4ccfebe-d99c-472f-8e2c-f38ad8a58e5e)